packages/runtime-core/src/componentPublicInstance.ts

```diff
@@ -290,7 +290,7 @@ export const PublicInstanceProxyHandlers: ProxyHandler<any> = {
     // access on a plain object, so we use an accessCache object (with null
     // prototype) to memoize what access type a key corresponds to.
     let normalizedProps
-    if (key[0] !== '$') {
+    if (key[0] !== '$' && key[0] !== '_') {
```

Comment:
this maybe a breaking change.

Reply:
There is a warning for users to avoid  such practice while using "this.$xxx" , and "this.$xxx" won't takes effect.
But "_xxx" doesn't. 
I consider "_xxx" was treat as a common property and it won't enter the conditional expressions for "reserved prefix".